package.json

```diff
@@ -8,13 +8,13 @@
   ],
   "dependencies": {
     "@edx/brand": "npm:@openedx/brand-openedx@1.2.2",
-    "@edx/frontend-component-footer": "12.2.0",
+    "@edx/frontend-component-footer": "12.5.2",
     "@edx/frontend-enterprise-catalog-search": "5.5.0",
     "@edx/frontend-enterprise-hotjar": "2.0.0",
     "@edx/frontend-enterprise-logistration": "4.0.5",
     "@edx/frontend-enterprise-utils": "4.0.5",
-    "@edx/frontend-platform": "5.0.0",
-    "@edx/paragon": "20.45.4",
+    "@edx/frontend-platform": "6.2.0",
+    "@edx/paragon": "21.5.6",
```

Comment:
Note: the breaking change between v20 and v21 involves an a11y fix for the `SelectableBox.Set` component, which this MFE uses in two places. One of those [usages](https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/96235426307642ef63cc75087df8911162a89965/src/components/skills-quiz-v2/JobCardComponent.jsx#L75) has not been accounted for in this upgrade as documented in the [release notes](https://github.com/openedx/paragon/releases/tag/v21.0.0).